python/src/Distribution_doc.i.in

```diff
@@ -30,6 +30,8 @@ OT_Distribution_computeSequentialConditionalQuantile_doc
 OT_Distribution_computeDDF_doc
 %feature("docstring") OT::Distribution::computeDensityGenerator
 OT_Distribution_computeDensityGenerator_doc
+%feature("docstring") OT::Distribution::computeLogDensityGenerator
+OT_Distribution_computeLogDensityGenerator_doc
```

Comment:
and remove this one

Reply:
I moved it into the EllipticalDistribution_doc.i.in